ios/XUMM/Libs/Security/Vault/VaultManager.m

```diff
@@ -189,7 +189,7 @@ + (BOOL) reKeyVault: (NSString *)vaultName oldKey:(NSString *)oldKey newKey:(NSS
   // try to create the new vault under a temp name with new key
   // with this we will make sure we are able to recover the key in case of failure
   NSString *recoveryVaultName = getRecoveryVaultName(vaultName);
-  [VaultManagerModule createVault:recoveryVaultName data:cleartext key:newKey];
+  [VaultManagerModule createVault:recoveryVaultName data:cleartext key:oldKey];
```

Comment:
😬😅

Reply:
Haha yeah, I forgot to change the comment